src/components/AppHeader/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react';
+import { WorkshopHeaderWrapper, TitlesWrapper } from './styled';
+import {
+  CurrRouteName,
+  LinkText,
+  PageSubTitle,
+  PageTitle,
+} from '../../typography';
+import { Breadcrumb } from '../../constants';
+import { LogoutButton } from '../LogoutButton';
+
+type Props = {
+  title: string;
+  subTitle: string;
+  routes: Breadcrumb[];
+};
+
+export const AppHeader: FC<Props> = ({ title, subTitle, routes }) => {
+  return (
+    <WorkshopHeaderWrapper>
+      <div>
+        <div>
+          {routes.map(({ path, breadcrumbName }, i) => {
+            if (i + 1 === routes.length)
+              return (
+                <CurrRouteName key={`breadcrumbName${i}`}>
```

Comment:
Do you want, maybe => `${breadcrumbName}${i}`

?

Reply:
no, but why not?
Fixed.